diffrax/_saveat.py

```diff
@@ -32,9 +32,29 @@ class SubSaveAt(eqx.Module):
     t0: bool = False
     t1: bool = False
     ts: Optional[Real[Array, " times"]] = eqx.field(default=None, converter=_convert_ts)
-    steps: bool = False
+    steps: int = 0
     fn: Callable = save_y
```

Comment:
We can remove the default values here now that we have an explicit `__init__` method instead.